Frameworks/UIKit/NSTextContainer.mm

```diff
@@ -64,8 +64,9 @@ - (NSArray*)exclusionPaths {
 
 /**
  @Status Caveat
-
- @Notes reaminingRect parameter is not supported
+ @Notes writingDirection and atIndex parameters are ignored
+ TODO::1123 This needs to handle multiple exclusion zones properly when creating remainingRect
+            Currently processing exclusion zones left-to-right fails to calculate properly
 */
 
 - (CGRect)lineFragmentRectForProposedRect:(CGRect)proposed
```

Comment:
nit: unnecessary line break here, not sure if it works with our annotation tools